include/inl/gnc_utilities.inl

```diff
@@ -0,0 +1,138 @@
+//
+// include/gnc_utilities.hpp
+// PSim
+//
+// Contributors:
+//   Kyle Krol  kpk63@cornell.edu
+//
+// Pathfinder for Autonomous Navigation
+// Space Systems Design Studio
+// Cornell Univeristy
+//
+
+#include "../gnc_utilities.hpp"
+
+namespace gnc {
+namespace utl {
+
+template <typename T>
+constexpr void quat_conj(lin::Vector<T, 4> const &q, lin::Vector<T, 4> &res) {
```

Comment:
Consider adding static_asserts that check if T is a float or double via `std::is_same<T, float>::value`. Unless lin already does this type checking?

Reply:
Lin already does that type checking. Currently it only enable the `lin::internal::traits<...>` struct if `T` is a float or double (only supported types currently).